### ISSUE TITLE
adi broken for SLE

### ISSUE THREAD

**Issue by coolo** [openSUSE/osc-plugin-factory#1113]
```
group 'factory-staging' not found
Traceback (most recent call last):
  File "/home/coolo/prod/osc/osc-wrapper.py", line 41, in <module>
    r = babysitter.run(osccli)
  File "/home/coolo/prod/osc/osc/babysitter.py", line 61, in run
    return prg.main(argv)
  File "/home/coolo/prod/osc/osc/cmdln.py", line 343, in main
    return self.cmd(args)
  File "/home/coolo/prod/osc/osc/cmdln.py", line 366, in cmd
    retval = self.onecmd(argv)
  File "/home/coolo/prod/osc/osc/cmdln.py", line 500, in onecmd
    return self._dispatch_cmd(handler, argv)
  File "/home/coolo/prod/osc/osc/cmdln.py", line 1230, in _dispatch_cmd
    return handler(argv[0], opts, *args)
  File "/home/coolo/.osc-plugins/osc-staging.py", line 596, in do_staging
    AdiCommand(api).perform(args[1:], move=opts.move, by_dp=opts.by_develproject, split=opts.split)
  File "/home/coolo/.osc-plugins/osclib/adi_command.py", line 136, in perform
    if self.api.is_user_member_of(self.api.user, 'factory-staging'):
  File "/home/coolo/.osc-plugins/osclib/stagingapi.py", line 1649, in is_user_member_of
    root = ET.fromstring(get_group(self.apiurl, group))
  File "src/lxml/lxml.etree.pyx", line 3213, in lxml.etree.fromstring (src/lxml/lxml.etree.c:78986)
  File "src/lxml/parser.pxi", line 1847, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:118298)
ValueError: can only parse strings
```

**Comment by jberry-suse** [comment]
This would appear to be solved by #1097.

**Comment by coolo** [comment]
Sorry for the noise - I was logged into the wrong computer with an outdated osc
